packages/sprotty/package.json

```diff
@@ -25,10 +25,9 @@
     "inversify": "~6.0.2"
   },
   "dependencies": {
-    "autocompleter": "^9.1.0",
+    "autocompleter": "^9.1.2",
     "file-saver": "^2.0.5",
-    "inversify": "~6.0.2",
```

Comment:
This is done intentionally, because yarn does not automatically install `peerDependencies`, I guess `inversify` is specified elsewhere, too right?

Reply:
npm and pnpm are ok with that when using a lib. yarn is not!

Reply:
@spoenemann when you remove the lock file (you get the warning). And once you remove other references to ìnversify` in the workspace (optional in elk or dependency in examples) the dependency is gone ⚠️ :
![image](https://github.com/eclipse-sprotty/sprotty/assets/21079138/887560bd-ced2-48f4-a2a3-33c9102268d9)

Reply:
`peerDependency` specifies the compatible version range. Important when using sprotty in another project where a conflicting version is in place.

With the change you suggest using sprotty elsewhere, npm an pnpm will fulfill a peerDependency if not met by other dependencies, yarn requires you to specify it. Therefore, specifying it here removes the problem for end-users. yarn behaving differently would be the better option, though.

Reply:
But then I would remove the `peerDependency` and just keep it as a `dependency`, reverting the change made in #357 (as proposed in #296).

Reply:
Let's do this. 👍